### PR TITLE
Dev router users

### PR DESCRIPTION
**Pull request by ian-dev**
fix: user data loading in user profile page

### CODE REVIEW

File: src/components/Table.js [hunk 1]
```diff
@@ -1,22 +1,40 @@
 import { useMemo, useState } from "react";
 import { useGetUsersQuery } from "../utils/api";
 import { DataGrid } from "@mui/x-data-grid";
-import GenericButton from "./GenericButton";
-import VisibilityIcon from "@mui/icons-material/Visibility";
-import RemoveCircleIcon from "@mui/icons-material/RemoveCircle";
+import { Button } from "@mui/material";
```
Comment (andresarenasv) [inline src/components/Table.js]: Hey Juli, please always use full path import for libraries such material ui, lodash, datefns

import Button from "@mui/material/Button";

**Review by andresarenasv — CHANGES_REQUESTED** [review]

File: src/components/Table.js [hunk 2]
```diff
@@ -1,22 +1,40 @@
...
-};
-
 export default function UsersTable() {
   const { data = USERS, isLoading, refetch } = useGetUsersQuery();
+  const userListHelper = useRef(USERS);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    userListHelper.current = data;
+    // console.log(data);
```
Comment (nassao) [inline src/components/Table.js]: Hey, doc!
Try not to have any commented code in your apps.
Use comments only for documentation purposes.

File: src/components/Table.js (hunk 1 above)
Comment (ian-dev) [reply src/components/Table.js]: Noted! However something funny happens, as soon as I enter the full path for the import the app crashes. I'm trying to figure out what's going on


Screenshot (1)

Comment (ian-dev) [reply src/components/Table.js]: Nevermind, I got what was wrong with the import statement

File: src/components/Table.js (hunk 2 above)
Comment (ian-dev) [reply src/components/Table.js]: Sure, noted

**Pull request merged**